BiSON.podspec

```diff
@@ -0,0 +1,77 @@
+#
+#  Be sure to run `pod spec lint BiSON.podspec' to ensure this is a
+#  valid spec and to remove all comments including this before submitting the spec.
+#
+#  To learn more about Podspec attributes see http://docs.cocoapods.org/specification.html
+#  To see working Podspecs in the CocoaPods repo see https://github.com/CocoaPods/Specs/
+#
+
+Pod::Spec.new do |s|
+
+  # ―――  Spec Metadata  ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――― #
+
+  s.name         = "BiSON"
+  s.version      = "1.0.0"
```

Comment:
For the sake of including both the Android and iOS `ports` under one version, I will likely be increasing the version to `1.0.1` after this is merged. Could you change this version accordingly?